cardano-api/src/Cardano/Api/Typed/Internal.hs

```diff
@@ -0,0 +1,10 @@
+module Cardano.Api.Typed.Internal
+  ( toNetworkMagic
+  ) where
+
+import           Cardano.Api.Typed.NetworkId
+import           Ouroboros.Network.Magic
+
+toNetworkMagic :: NetworkId -> NetworkMagic
+toNetworkMagic  Mainnet     = NetworkMagic 764824073
+toNetworkMagic (Testnet nm) = nm
```

Comment:
We should not just be adding yet another copy of this magic number.

Please use `Cardano.Chain.Genesis.Data..mainnetProtocolMagicId` with something like (warning unchecked):
```
toNetworkMagic  Mainnet     = NetworkMagic $ Byron.unProtocolMagicId Byron.mainnetProtocolMagicId
```

Reply:
I didn't do it this way because `‘cardano-ledger-0.1.0.0` wasn't a dependency, but I can add the dependency.

Reply:
Oh wait, I can't.  The module is not public.

Reply:
I think you can import it from [`Cardano.Chain.Genesis`](Cardano.Chain.Genesis).

Reply:
Yep.  Thanks!

Reply:
Fixed!